docs/azure/sdk/authentication-azure-hosted-apps.md

```diff
@@ -0,0 +1,110 @@
+
+## Managed identity types
+
+There are two types of managed identities:
+
+- **System-assigned managed identities** - This type of managed identity is provided by and tied directly to an Azure resource.  When you enable managed identity on an Azure resource, you get a system-assigned managed identity for that resource.  A system-assigned managed identity is tied to the lifecycle of the Azure resource it's associated with. When the resource is deleted, Azure automatically deletes the identity for you.  Since all you have to do is enable managed identity for the Azure resource hosting your code, this is the easiest type of managed identity to use.
+- **User-assigned managed identities** - You may also create a managed identity as a standalone Azure resource. This is most frequently used when your solution has multiple workloads that run on multiple Azure resources that all need to share the same identity and same permissions.  For example, if your solution had components that ran on multiple App Service and virtual machine instances that all needed access to the same set of Azure resources, creating and using a user-assigned managed identity across those resources would make sense.
+
+This article will cover the steps to enable and use a system-assigned managed identity for an app.  If you need to use a user-assigned managed identity, see the article [Manage user-assigned managed identities](/azure/active-directory/managed-identities-azure-resources/how-manage-user-assigned-managed-identities) to see how to create a user-assigned managed identity.
+
+## 1 - Enable managed identity in the Azure resource hosting the app
+
+The first step is to enable managed identity on Azure resource hosting your app.  For example, if you're hosting a .NET application using Azure App Service, you need to enable managed identity for the App Service web app that is hosting your app.  If you were using a virtual machine to host your app, you would enable your VM to use managed identity.
+
+You can enable managed identity to be used for an Azure resource using either the Azure portal or the Azure CLI.
+
+### [Azure portal](#tab/azure-portal)
+
+| Instructions    | Screenshot |
+|:----------------|-----------:|
+| [!INCLUDE [Enable managed identity step 1](<./includes/enable-managed-identity-azure-portal-1.md>)] | :::image type="content" source="./media/enable-managed-identity-azure-portal-1-240px.png" alt-text="A screenshot showing how to use the top search bar in the Azure portal to locate and navigate to a resource in Azure." lightbox="./media/enable-managed-identity-azure-portal-1.png"::: |
```

Comment:
@alexwolfmsft What is the reason to use the angle brackets around the path to the include file?